tools/istio-docker.mk

```diff
@@ -25,7 +25,7 @@ docker: docker.all
 
 DOCKER_TARGETS ?= docker.pilot docker.proxyv2 docker.app docker.app_sidecar_ubuntu_xenial \
 docker.app_sidecar_ubuntu_bionic docker.app_sidecar_ubuntu_focal docker.app_sidecar_debian_9 \
-docker.app_sidecar_debian_10 docker.test_policybackend docker.mixer docker.mixer_codegen \
+docker.app_sidecar_debian_10 docker.app_sidecar_centos docker.test_policybackend docker.mixer docker.mixer_codegen \
```

Comment:
Do we also need to add the targets to prow/lib.sh?

Reply:
Yes, good catch